src/core/lib/channel/promise_based_filter.h

```diff
@@ -373,6 +427,11 @@ MakeFilterCall(Derived* derived) {
 // - absl::Status $INTERCEPTOR_NAME($VALUE_TYPE&):
 //   the filter intercepts this event, and can modify the value.
 //   it can fail, in which case the call will be aborted.
+// - ServerMetadataHandle $INTERCEPTOR_NAME($VALUE_TYPE&)
```

Comment:
I just noticed that all of these parameters are being passed in as references, even though they're mutable.  I realize that style guide has been changed to say this is allowable, but I do still find it convenient to see mutable parameters passed in as a pointers, just so that there's a clear indicator in the caller that it's mutable.

If I'm the only one with this preference, then feel free to ignore this.